File: Java/Program/Baju.java

class Baju extends Aksesoris {
    private String untuk;
    private String size;
    private String merk;

    // constructors
    public Baju() {
        super();
        this.untuk = "";
        this.size = "";
        this.merk = "";
    }

    public Baju(int id, String namaProduk, double hargaProduk, int stokProduk, String jenis, String bahan, String warna,
            String untuk, String size, String merk) {
        super(id, namaProduk, hargaProduk, stokProduk, jenis, bahan, warna);
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    // getters
    public String getUntuk() {
        return untuk;
    }

    public String getSize() {
        return size;
    }

    public String getMerk() {
        return merk;
    }

    // setters
    public void setUntuk(String untuk) {
        this.untuk = untuk;
    }

    public void setSize(String size) {
        this.size = size;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }
}

File: Python/Program/Baju.py
from Aksesoris import Aksesoris


class Baju(Aksesoris):
    def __init__(self, id=0, nama_produk="", harga_produk=0.0, stok_produk=0, jenis="", bahan="", warna="", untuk="", size="", merk=""):
        super().__init__(id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna)
        self.untuk = untuk
        self.size = size
        self.merk = merk

    # getters
    def get_untuk(self):
        return self.untuk

    def get_size(self):
        return self.size

    def get_merk(self):
        return self.merk

    # setters
    def set_untuk(self, untuk):
        self.untuk = untuk

    def set_size(self, size):
        self.size = size

    def set_merk(self, merk):
        self.merk = merk
